### PR TITLE
duration rounding fixed

### LINKED ISSUE #487: Durations rounding is misleading
See:
image

Duration should be: 3 hours + 17 mn + 2 hours... but it is 3 hours. Something is wrong here because ZIM files are here not build in paralell.

### PR DESCRIPTION
**Pull request by mahakporwal02**
#487 Duration rounding is misleading
## Changes
I have added a new function using npm package to show duration

### CODE REVIEW

**Review by rgaudin — CHANGES_REQUESTED** [review]
Thank you but you're adding an external library without including it… can't work

``` diff
diff --git i/dispatcher/frontend-ui/package.json w/dispatcher/frontend-ui/package.json
index b0bf560b..648a494b 100644
--- i/dispatcher/frontend-ui/package.json
+++ w/dispatcher/frontend-ui/package.json
@@ -17,6 +17,7 @@
     "core-js": "^3.3.2",
     "deep-diff": "^1.0.2",
     "filesize": "^6.0.1",
+    "humanize-duration": "^3.24.0",
     "jsonwebtoken": "^8.5.1",
     "moment": "^2.24.0",
     "password-generator": "^2.2.3",
diff --git i/dispatcher/frontend-ui/yarn.lock w/dispatcher/frontend-ui/yarn.lock
index e54036b9..85a5da98 100644
--- i/dispatcher/frontend-ui/yarn.lock
+++ w/dispatcher/frontend-ui/yarn.lock
@@ -4554,6 +4554,11 @@ human-signals@^1.1.1:
   resolved "https://registry.yarnpkg.com/human-signals/-/human-signals-1.1.1.tgz#c5b1cd14f50aeae09ab6c59fe63ba3395fe4dfa3"
   integrity sha512-SEQu7vl8KjNL2eoGBLF3+wAjpsNfA9XMlXAYj/3EdaNfAlxKthD1xjEQfGOUhllCGGJVNY34bRr6lPINhNjyZw==

+humanize-duration@^3.24.0:
+  version "3.24.0"
+  resolved "https://registry.yarnpkg.com/humanize-duration/-/humanize-duration-3.24.0.tgz#d851ca6fb93277e30ad789e88bd2019655f514c1"
+  integrity sha512-B3udnqisaDeRsvUSb+5n2hjxhABI9jotB+i1IEhgHhguTeM5LxIUKoVIu7UpeyaPOygr/Fnv7UhOi45kYYG+tg==
+
 iconv-lite@0.4.24, iconv-lite@^0.4.24:
   version "0.4.24"
   resolved "https://registry.yarnpkg.com/iconv-lite/-/iconv-lite-0.4.24.tgz#2022b4b25fbddc21d2f524974a474aafe733908b"
```

**Comment by mahakporwal02** [comment]
@rgaudin sorry I am new to open source might be due to branching mistake

File: dispatcher/frontend-ui/src/constants.js [hunk 1]
```diff
@@ -15,8 +16,22 @@ function format_duration(diff) { // display a duration in a standard format
...
+          y: () => "y",
+          mo: () => "mo",
+          w: () => "w",
+          d: () => "d",
+          h: () => "h",
+          m: () => "m",
+          s: () => "s",
+          ms: () => "ms",
+        },
+      },largest: 2 ,round : true ,delimiter: " "});
```
Comment (rgaudin) [inline dispatcher/frontend-ui/src/constants.js:32]: Please use

``` js
}, largest: 2, round: true , delimiter: " ", spacer: ""});
```
so we keep our code formatting and adjust the displayed string as well.

**Review by rgaudin — APPROVED** [review]
Great! Please fix the indicated line and I'll merge.

**Comment by mahakporwal02** [comment]
@rgaudin changes done

**Review by rgaudin — APPROVED** [review]

**Pull request merged**